src/whylogs/songbird/wrapper.py

```diff
@@ -0,0 +1,45 @@
+from google.protobuf.message import Message
+import whylabs_client
+from whylabs_client.api import sessions_api
+import requests
+
+configuration = whylabs_client.Configuration( host = "https://songbird.development.whylabsdev.com" )
+
+session_token = None
+
+def get_songbird_client():
+    with whylabs_client.ApiClient(configuration) as api_client:
+        return sessions_api.SessionsApi(api_client)
+
+def start_session():
+    global session_token
+    client = get_songbird_client()
+    try:
+        response = client.create_session()
+        session_token = response.get("token")
+    except Exception as e:
+        print(f"Failed to create songbird session: {e}")
```

Comment:
Don't use `printf`. Use `logger` API instead

Reply:
Replaced debug logging with `logger`, but what about printing the URL at the end of a session?